ush/wafs/evs_wafs_atmos_stats_preparedata.sh

```diff
@@ -103,12 +103,12 @@ if [ -z $FHOURS_EVSlist ] ; then
 else
     if [ ! -z "$missingForecast" ] ; then
 	if [[ $SENDMAIL = YES ]] ; then
-            export subject="Some forecast files are missing for EVS ${COMPONENT}"
-            echo "WARNING: Some $CENTER forecasts $missingForecast are missing for $OBSERVATION valid date ${VDATE}${cc}. METplus continues." > mailmsg.$OBSERVATION.$CENTER.$RESOLUTION
+            export subject="A subset of $CENTER forecast files is missing for EVS ${COMPONENT}"
+            echo "WARNING: A subset of $CENTER forecasts ($missingForecast) is missing for $OBSERVATION valid date ${VDATE}${cc}. METplus continues." > mailmsg.$OBSERVATION.$CENTER.$RESOLUTION
             echo "Job ID: $jobid" >> mailmsg.$OBSERVATION.$CENTER.$RESOLUTION
 	    cat mailmsg.$OBSERVATION.$CENTER.$RESOLUTION | mail -s "$subject" $MAILTO
 	fi
-	echo "WARNING: Some $CENTER forecasts $missingForecast are missing for $OBSERVATION valid date ${VDATE}${cc}. METplus continues."
+	echo "WARNING: A subset of $CENTER forecasts ($missingForecast) is missing for $OBSERVATION valid date ${VDATE}${cc}. METplus continues."
```

Comment:
This echo statement is repeated outside the SENDMAIL block. I thought the consolidated WARNING statements were only going in the email block. If so, and if we still have the warning statements that list the individual files that are missing, can this repeated consolidated statement be removed? @YaliMao-NOAA @PerryShafran-NOAA